bin/oref0-determine-basal.js

```diff
@@ -20,81 +20,70 @@ var determine_basal = require('oref0/lib/determine-basal/determine-basal');
 /* istanbul ignore next */
 if (!module.parent) {
     var argv = require('yargs')
-      .usage("$0 iob.json currenttemp.json glucose.json profile.json [[--auto-sens] autosens.json] [meal.json] [--reservoir reservoir.json] --currentTime [currentTime]")
+      .usage("$0 <iob.json> <currenttemp.json> <glucose.json> <profile.json> [--auto-sens <autosens.json>] [--meal <meal.json>] [--reservoir <reservoir.json>] [--currentTime <currentTime>]")
       .option('auto-sens', {
         alias: 'a',
+        nargs: 1,
         describe: "Auto-sensitivity configuration",
-        default: true
-
+        default: false
```

Comment:
What do these default changes do?

Reply:
Defaulting it to true causes determine_basal to crash if no argument was passed in because it was true vs. undefined.

Default it to false allows determine_basal to test for whether it's defined or false to determine if a valid value was provided.